### PR TITLE
Update the find command to support search by freetime at a specified …

### LINKED ISSUE #73: Update find command to support search by freetime at a specified day

### PR DESCRIPTION
**Pull request by LuoZYi**
…day and update user guide for find and editft command

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java [hunk 1]
```diff
@@ -57,11 +62,24 @@ public FindCommand parse(String args) throws ParseException {
...
-        if (argMultimap.getValue(PREFIX_FROM).isPresent() && argMultimap.getValue(PREFIX_TO).isPresent()) {
-            FreeTime freeTime = ParserUtil.parseFreeTime(argMultimap.getValue(PREFIX_FROM).orElse(null),
-                    argMultimap.getValue((PREFIX_TO)).orElse(null));
-            // since TimeInterval is same for all days for now, we just use the first day
-            predicates.add(new AvailableTimePredicate(freeTime.getDay(0)));
+        if (argMultimap.getValue(PREFIX_DAY).isPresent()
+                && argMultimap.getValue(PREFIX_FROM).isPresent()
+                    && argMultimap.getValue(PREFIX_TO).isPresent()) {
+            try {
+                Integer dayOfWeek = Integer.parseInt(argMultimap.getValue(PREFIX_DAY).orElse(null));
```
Comment (PohSayKeong) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java]: dont have to orElse here as you have already checked if it is present

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java [hunk 2]
```diff
@@ -57,11 +62,24 @@ public FindCommand parse(String args) throws ParseException {
...
+                if (dayOfWeek < 1 || dayOfWeek > 5) {
+                    throw new ParseException(EditFreeTimeCommand.MESSAGE_INVALID_DAY);
+                }
+                FreeTime freeTime = ParserUtil.parseFreeTime(argMultimap.getValue(PREFIX_FROM).orElse(null),
```
Comment (PohSayKeong) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java]: this is unused

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java (hunk 1 above)
Comment (PohSayKeong) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java]: can use a primitive here

File: src/main/java/seedu/address/model/person/predicates/AvailableTimePredicate.java [hunk 3]
```diff
@@ -8,29 +8,43 @@
  * Tests that a {@code Person}'s {@code FreeTime} is within the FreeTime that is given.
  */
 public class AvailableTimePredicate implements FindCommandPredicate {
+    private static final String[] DAY_OF_WEEK = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday"};
```
Comment (PohSayKeong) [inline src/main/java/seedu/address/model/person/predicates/AvailableTimePredicate.java]: Consider using something like this `DayOfWeek.of(i + 1).getDisplayName(TextStyle.SHORT, Locale.ENGLISH);`

File: src/main/java/seedu/address/model/person/predicates/AvailableTimePredicate.java [hunk 4]
```diff
@@ -8,29 +8,43 @@
...
         }
-        return person.getFreeTime().getIntervals().stream()
-                .anyMatch(personInterval -> {
-                    if (personInterval != null) {
-                        return personInterval.isInBetween(interval);
-                    }
-                    return false;
-                });
+
+        TimeInterval timeInterval = person.getFreeTime().getIntervals().get(day - 1);
```
Comment (PohSayKeong) [inline src/main/java/seedu/address/model/person/predicates/AvailableTimePredicate.java]: use `getDay` here

**Review by PohSayKeong — CHANGES_REQUESTED** [review]
Minor changes required, thanks!

**Comment by LuoZYi** [comment]
Changed as suggested.

**Review by PohSayKeong — APPROVED** [review]
LGTM 
Closes #73

**Pull request merged**